### PR TITLE
Add permission for admin students for New Labor Status Form

### PR DESCRIPTION
**Pull request by stevensonmichel**
## Issue: 
Previously, students who are also Labor Admins did not have access to the New Labor Status Form page. That was considered as a bug given that these students have the permissions to add submit labor status forms for students. 

## Solution: 
Through exploration, we discovered that in the backend such a logic to allow labor admin students to see the New Labor Status Form was already implemented. However, the sidebar was not displaying the New Labor Status Form for these students. So, we made changes there to reflect the privileges of accessibility for these students. 

## Test:
Assuming you're in the LSF directory
- source setup.sh
- database/reset_database.sh from-backup
- change the login username into a student who is a Labor Admin (ex. floresd)
- export FLASK_ENV=staging
- flask run
Now open the application and verify if New Labor Status Form is present as an option in the sidebar. 

Fixes Issue #1321
https://github.com/BCStudentSoftwareDevTeam/celts/issues/1321

### CODE REVIEW

File: app/templates/sidebar.html [hunk 1]
```diff
@@ -39,6 +39,14 @@ <h4>Student Search</h4>
               </a>
           </div>
         </div>
+      {% elif currentUser.student and currentUser.isLaborAdmin %}
```
Comment (bledsoef) [inline app/templates/sidebar.html]: Could we potentially add this conditional to the other New Labor Status Form instead of duplicating all this code?

**Review by bledsoef — CHANGES_REQUESTED** [review]
Functionality looks alright besides some tracy problems which are unrelated. Just might try to prevent duplicating code. Nice job!

File: app/templates/sidebar.html (hunk 1 above)
Comment (hoerstl) [reply app/templates/sidebar.html]: Yessir. I've replaced the single if statement above with three so the sidebar components remain in order and removed the duplicate code.

**Review by bledsoef — APPROVED** [review]
LGTM

**Pull request merged**